### PR TITLE
[C3] Updating README and bumping version

### PR DESCRIPTION
**Pull request by jculvey**
**What this PR solves / how to test:**

Updating the README ahead of the release to be clear and concise. It's a little spartan for now, but we can improve it in the near future.

Also includes a one time version bump for the first release. After this, we'll manage version and publishes via changesets.

**Associated docs issue(s)/PR(s):**

- [insert associated docs issue(s)/PR(s)]

**Author has included the following, where applicable:**

- [ ] Tests
- [ ] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))

**Reviewer is to perform the following, as applicable:**

- Checked for inclusion of relevant tests
- Checked for inclusion of a relevant changeset
- Checked for creation of associated docs updates
- Manually pulled down the changes and spot-tested

### CODE REVIEW

File: packages/create-cloudflare/package.json [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-		"vite-tsconfig-paths": "^4.0.8",
-		"vitest": "^0.30.0",
-		"which-pm-runs": "^1.1.0",
-		"wrangler": "^2.13.0",
-		"yargs": "^17.7.1"
-	}
-}
+  "name": "create-cloudflare",
+  "version": "2.0.1",
+  "description": "",
```
Comment (admah) [inline packages/create-cloudflare/package.json:4]:
```suggestion
  "description": "A CLI for creating and deploying new applications to Cloudflare.",
```

File: packages/create-cloudflare/package.json [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+  "license": "ISC",
+  "author": "",
```
Comment (admah) [inline packages/create-cloudflare/package.json:6]: Should we have something here like `wrangler@cloudflare.com`?

**Review by admah — APPROVED** [review]
LGTM! Added a couple comments.

File: packages/create-cloudflare/README.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-`create-cloudflare` is a scaffolding tool meant to help users get started with Cloudflare Pages using their framework of choice as quickly and easily as possible.
-
-Instead of having to choose from a set of pre-constructed templates, this tool allows you to run the creation commands of your desired framework directly. This means that you can configure it with all the options the
-
-To get started, run the following command:
+A CLI for creating and deploying new applications to Cloudflare.
 
 ```bash
-npm create cloudflare
+npm create cloudflare@latest
```
[the remark this answers is not in the document]
Comment (jculvey) [reply packages/create-cloudflare/README.md:6]: I think we should -- I've noticed that all the other scaffolding tools we've looked at suggest the same, so we should probably follow suit. I'm pretty sure `npm create` will cache the package as a global install, and if we don't plan to do major bumps very often people could end up with pretty stale versions.

[the remark this answers is not in the document]
Comment (petebacondarwin) [reply packages/create-cloudflare/README.md:6]: Yeah otherwise it can use what is in the local cache, which might be outdated.

**Review by petebacondarwin — APPROVED** [review]

**Pull request merged**